src/aframe-streetmix-parsers.js

```diff
@@ -850,33 +866,26 @@ function processBuildings (left, right, streetWidth, showGround, length) {
 
     if (currentValue === 'waterfront') {
       const objectPositionX = positionX - (sideMultiplier * buildingLotWidth / 2);
-
       const placedObjectEl = document.createElement('a-entity');
       placedObjectEl.setAttribute('class', 'seawall-parent');
-      placedObjectEl.setAttribute('position', objectPositionX + ' 0 10'); // position="1.043 0.100 -3.463"
+      placedObjectEl.setAttribute('position', objectPositionX + ' 0 0'); // position="1.043 0.100 -3.463"
       placedObjectEl.classList.add('seawall-parent-' + side);
-      // add the new elmement to DOM
       buildingElement.appendChild(placedObjectEl);
-
       // clone a bunch of seawalls under the parent
       const rotationCloneY = (side === 'right') ? -90 : 90;
-      cloneMixinAsChildren({ objectMixinId: 'seawall', parentEl: placedObjectEl, rotation: '0 90 0' + rotationCloneY + ' 0', step: 15, radius: clonedObjectRadius });
+      cloneMixinAsChildren({ objectMixinId: 'seawall', parentEl: placedObjectEl, rotation: '0 ' + rotationCloneY + ' 0', step: 15, radius: 0.45*length });
```

Comment:
good catch, the rotation value set in the old line 863 was invalid (resulting in vec5 instead of vec3)